pkg/gator/errors.go

```diff
@@ -54,4 +54,6 @@ var (
 	// ErrNilOldObject indicates that the AdmissionRequest did not provide an oldObject.
 	// Gatekeeper expects oldObject to be non nil on DELETE operations.
 	ErrNilOldObject = errors.New("oldObject is nil")
+	// ErrInvalidYAML indicates that a .yaml/.yml file was not parseable.
+	ErrInvalidYAML = errors.New("invalid yaml")
```

Comment:
```suggestion
	ErrInvalidYAML = errors.New("cannot parse yaml file")
```

Reply:
I believe this error text is just refactored existing code, we should preserve the existing text for consistency.